<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Controller v1.0</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonRun">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>771</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputIntPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputExtPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Зупинка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRestart">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Перезапуск</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExternalPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зовнішній порт:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInternalPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Внутрішній порт:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hostIPList">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>121</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>20</height>
     </size>
    </property>
    <property name="maxVisibleItems">
     <number>15</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelServerIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Адреса сервера:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionRestart"/>
    <addaction name="actionSaveCfg"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="title">
     <string>Сервіс</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuService"/>
  </widget>
  <action name="actionRun">
   <property name="text">
    <string>Запустити</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Зупинити</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Перезапустити</string>
   </property>
  </action>
  <action name="actionSaveCfg">
   <property name="text">
    <string>Зберегти налаштування</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Завантажити налаштування</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Вийти</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Допомога</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
